doc/Customization-guide/How-to-contribute/Jira-flow/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Jira flow
+permalink: /:path/
+sort_idx: 20
+---
+
+* TOC
+{:toc}
+
+{% include variables.md %}
+
+The Kaa project team uses [Atlassian Jira](https://www.atlassian.com/software/jira) for issue tracking in the development process.
+This section describes when and how you can create Jira issues at [http://jira.kaaproject.org/](http://jira.kaaproject.org/) if you decided to contribute to Kaa project.
+
+## When to create issues
+
+To decide whether you should create a Jira issue, use [Kaa Stack Overflow](http://stackoverflow.com/questions/tagged/kaa) to consult with Kaa team and find out whether your issue:
+
+* Is indeed an issue by nature.
+* Is reproducible.
+* Has not been spotted by other contributors yet.
+
+If you confirmed all above statements to be true, proceed to the next section to find out which information you should provide when creating a Jira issue.
+
+## Information to provide
+
+To create a Jira issue, go to [http://jira.kaaproject.org/](http://jira.kaaproject.org/), click the **Create** button.
+In the **Create Issue** pop-up, provide the following information about the issue:
+
+| Field | Description |
+|-------|-------------|
+| Project | Select the project from the drop-down list depending on which [Kaa project repository](https://github.com/kaaproject) the issue is related to. When in doubt, select **Kaa**. |
+| Issue Type | See [Issue types](#issue-types) to make the right choice. When in doubt, select **Task**. |
+| Summary | Provide a one-line summary of the issue in clear and concise language. |
+| Component/s | The drop-down list of this field depends on the project you selected in the **Project** field. |
+| Affects Version/s | Select the version affected by the issue. |
+| Description | Provide a meaningful description of the issue with all the details you find relevant. Describe the steps to reproduce the issue, and the environment in which you found it. In general, feel free to share any information you think is related to the issue. See also [Issue types](#issue-types) for specific information to be provided depending on the issue type. |
+| Attachment | Feel free to attach images, documents and other files you think can help understand the issue-related information. |
+
+>**NOTES:** Please do not fill any other fields, they will be managed by the Kaa team members.
```

Comment:
As discussed with Jira admin, these permissions will be configured after we migrate to Jira 7 which is planned to be done at the end of the current sprint.